src/index.ts

```diff
@@ -77,7 +88,7 @@ function mapScalar(data: any, path: PropertyKey[], map: ScalarMapping) {
     if (Array.isArray(newSubData[segment])) {
       const subPath = path.slice(index + 1);
       newSubData[segment] = newSubData[segment].map((subData: unknown) =>
-        mapScalar(subData, subPath, map)
+        mapScalar(subData, subPath, mapping)
```

Comment:
Not a fault in either PR, but note that the renaming of `map` to `mapping` here would need to be resolved with this one in #23 if both get merged:
https://github.com/clentfort/urql-custom-scalars-exchange/pull/25/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R73

When merging the two locally I was surprised to find tests fail, and it was just that tiny little thing.